modules/installation/proc-install-rhdh-osd-gcp-operator.adoc

```diff
@@ -0,0 +1,34 @@
+// Module included in the following assemblies:
+// assembly-install-rhdh-osd-gcp.adoc
+
+[id="proc-install-rhdh-osd-gcp-operator_{context}"]
+= Installing {product} on {osd-short} on {gcp-short} using the Operator
+
+You can install {product-short} on {osd-short} on {gcp-short} using the {product} Operator.
+
+.Prerequisites
+* You have a valid {gcp-short} account.
+* Your {osd-short} cluster is running on {gcp-short}. For more information, see link:https://docs.redhat.com/en/documentation/openshift_dedicated/4/html/installing_accessing_and_deleting_openshift_dedicated_clusters/osd-creating-a-cluster-on-gcp[Creating a cluster on GCP] in {osd-brand-name} documentation.
+* You have administrator access to {osd-short} cluster and {gcp-short} project.
+
+.Procedure
```

Comment:
procedure is same as OCP, you may put that as well, although this is correct.

Reply:
I think it would be better to keep the information on this page rather than redirecting the user.